x-pack/plugins/gis/public/store/map.js

```diff
@@ -119,12 +119,15 @@ export function map(state = INITIAL_STATE, action) {
         }
       };
     case SET_GOTO:
+      const currentGoto = state.goto ? state.goto : {};
       return {
         ...state,
         goto: {
+          ...currentGoto,
           lat: action.lat,
           lon: action.lon,
           zoom: action.zoom,
+          bounds: action.bounds
```

Comment:
- split this up into two action-types/action-creators: goto-center (centerlonlat+zoom), goto-bounds (bounds)
- retain single `goto`-object in the store. (so we don't have multiple conflicting states).

cc @nreese